src/UserInterface/Image/createGradientOpacitySlider.js

```diff
@@ -27,9 +28,9 @@ function createGradientOpacitySlider(store, uiContainer) {
     store.imageUI.representationProxy.setEdgeGradient(gradientOpacity)
     store.renderWindow.render()
   }
-  autorun(() => {
-    updateGradientOpacity()
-  })
+  reaction(() => {
+    return store.imageUI.gradientOpacity
+  }, macro.debounce(updateGradientOpacity, 25, false))
```

Comment:
Could this be causing the lag we were observing the gradient opacity slider or this helping to address it?

Reply:
For me it was great improved by adding this debounce.  In fact, all the sliders were really laggy until I debounced them in a similar fashion.